.gitignore

```diff
@@ -3,4 +3,5 @@ __pycache__
 .hypothesis/
 build/
 reports/
-node_modules/
\ No newline at end of file
+node_modules/
+.DS_Store
```

Comment:
Is this necessary? I usually ask for editor-specific files for people to put in their global git ignore

Reply:
It is in fact specific to mac os x. I added it as I thought this may sneak in, as I use a remote dev box and my laptop, I will remove it in the updated patch. 

Brings up a great point, an official style guide and coding conventions would be great.